### PR TITLE
cod: unstable-2020-09-10 -> 0.1.0

### PR DESCRIPTION
**Pull request by kidonng**
###### Motivation for this change

https://github.com/dim-an/cod/releases/tag/v0.1.0

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 129951` at 44ff6866 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/02be8eb3f6d40a270585d4a2a363bed0">cod</a></li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

File: pkgs/tools/misc/cod/default.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
-  version = "unstable-2020-09-10";
+  version = "0.1.0";
 
   src = fetchFromGitHub {
     owner = "dim-an";
     repo = pname;
-    rev = "ae68da08339471dd278d6df79abbfd6fe89a10fe";
-    sha256 = "1l3gn9v8dcy72f5xq9hwbkvkis0vp4dp8qyinsrii3acmhksg9v6";
+    rev = "v${version}";
+    sha256 = "0h95y2pci3czwn9dh2xblq5cah7znx526khnla34rkxghkrdd3ix";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/cod/default.nix:11]:
```suggestion
    sha256 = "0wi680sxpv0kp1ggy21qp4c4ms79hw4z9w9kvp278p8z3y8wwglr";
```

File: pkgs/tools/misc/cod/default.nix [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
   src = fetchFromGitHub {
     owner = "dim-an";
     repo = pname;
-    rev = "ae68da08339471dd278d6df79abbfd6fe89a10fe";
-    sha256 = "1l3gn9v8dcy72f5xq9hwbkvkis0vp4dp8qyinsrii3acmhksg9v6";
+    rev = "v${version}";
+    sha256 = "0wi680sxpv0kp1ggy21qp4c4ms79hw4z9w9kvp278p8z3y8wwglr";
   };
 
   vendorSha256 = "1arllkiz1hk12hq5b2zpg3f8i9lxl66mil5sdv8gnhflmb37vbv3";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/cod/default.nix:14]:
```suggestion
  vendorSha256 = "0ann1fbh8rqys3rwbz5h9mfnvkpqiw5rgkd4c30y99706h2dzv4i";
```

**Comment by kidonng** [comment]
@SuperSandro2000 Thanks! Somehow my local build doesn't report the wrong SHA even though I was expecting vendorsha256 changes.

**Comment by SuperSandro2000** [comment]
You need to invalidate it otherwise it will just keep using the old one.

**Pull request merged**